src/json/seshistory.json

```diff
@@ -1,20 +1,36 @@
 {
```

Comment:
1. Is there a way to store these files somewhere that isn't in `src/`? Storing these files right next to the source code of the application is kind of dangerous.

2. This file shouldn't be tracked with version control -- having it here means that users will download the application and see "history" for sessions they didn't do. Could you remove this JSON file from the repository and add this file to the repository `.gitignore`? I know that a version of this file is already in the repository, so if you'd like we can hold off on this for now.

Reply:
I believe it can be moved outside src? Maybe a separate folder for userdata?
as for 2. yeah sure I agree. Will do after I finished one final tomorrow.

Reply:
Yeah, storing stuff in a separate folder (maybe named `.userdata/` or something) would work great. We could also add that directory to the `.gitignore`, which would make item 2 easier.

Reply:
Oh, also, good luck with your final btw! Sorry to bother you the night before with this :grimacing: